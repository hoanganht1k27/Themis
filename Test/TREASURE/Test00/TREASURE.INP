6 16
80 70 20 43 75 69
2 3
2 3
3 2
4 6
1 3
6 1
1 6
5 2
5 1
2 6
5 1
6 1
4 1
2 1
5 4
6 3
2 4 15 7 11 6 1 3 12 9 5 14 16 10 8 13
