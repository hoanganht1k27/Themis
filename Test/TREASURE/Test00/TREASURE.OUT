357
357
357
357
357
357
357
357
357
357
337
294
225
155
80
0
